package.json

```diff
@@ -59,30 +59,30 @@
   },
   "devDependencies": {
     "@google-cloud/storage": "^1.6.0",
-    "@material/button": "^0.41.0",
-    "@material/card": "^0.41.0",
-    "@material/checkbox": "^0.41.0",
-    "@material/chips": "^0.41.0",
-    "@material/drawer": "^0.41.0",
-    "@material/fab": "^0.41.0",
-    "@material/floating-label": "^0.41.0",
-    "@material/icon-button": "^0.41.0",
+    "@material/button": "^0.43.0",
+    "@material/card": "^0.43.0",
+    "@material/checkbox": "^0.43.0",
+    "@material/chips": "^0.43.0",
+    "@material/drawer": "^0.43.0",
+    "@material/fab": "^0.43.0",
+    "@material/floating-label": "^0.43.0",
+    "@material/icon-button": "^0.43.0",
     "@material/layout-grid": "^0.41.0",
-    "@material/line-ripple": "^0.41.0",
-    "@material/linear-progress": "^0.41.0",
-    "@material/list": "^0.41.0",
-    "@material/menu-surface": "^0.41.0",
-    "@material/notched-outline": "^0.41.0",
-    "@material/ripple": "^0.41.0",
-    "@material/select": "^0.40.1",
-    "@material/switch": "^0.41.0",
-    "@material/tab": "^0.41.0",
-    "@material/tab-bar": "^0.41.0",
-    "@material/tab-indicator": "^0.41.0",
-    "@material/tab-scroller": "^0.41.0",
-    "@material/textfield": "^0.41.0",
-    "@material/top-app-bar": "^0.41.0",
-    "@material/typography": "^0.41.0",
+    "@material/line-ripple": "^0.43.0",
+    "@material/linear-progress": "^0.43.0",
+    "@material/list": "^0.43.0",
+    "@material/menu-surface": "^0.43.0",
+    "@material/notched-outline": "^0.43.0",
+    "@material/ripple": "^0.43.0",
+    "@material/select": "^0.43.0",
```

Comment:
please revert select, textfield, notched outline, line ripple, and floating label. These are not going to work out of the box and will require a lot of updating (select and notched outline are refactored).